### PR TITLE
2027 cms iii

### PR DESCRIPTION
**Pull request by dczulada**
Pull requests into cqm-reports require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: lib/qrda-export/catIII/_header.mustache [hunk 1]
```diff
@@ -3,20 +3,26 @@
...
-  <!-- QRDA Category III Report - CMS (V9) -->
-  <templateId root="2.16.840.1.113883.10.20.27.1.2" extension="2024-12-01"/>
-  {{/ry2025_submission?}}
-  {{^ry2025_submission?}}
-  <!-- QRDA Category III Report - CMS (V7) -->
-  <templateId root="2.16.840.1.113883.10.20.27.1.2" extension="2022-12-01"/>
-  {{/ry2025_submission?}}
-{{/ry2026_submission?}}
+{{#ry2027_submission?}}
+  <!-- QRDA Category I Report - CMS (V11) -->
```
Comment (elsaperelli) [inline lib/qrda-export/catIII/_header.mustache]: Should this be Category III?

**Review by elsaperelli — CHANGES_REQUESTED** [review]
One small comment

**Review by elsaperelli — APPROVED** [review]
